### PR TITLE
notebook changes

### PR DESCRIPTION
**Pull request by benntay1**

### CODE REVIEW

File: notebooks/csrnet-training.ipynb [hunk 1]
```diff
@@ -16,6 +16,8 @@
     "import torch\n",
```
Comment (teojunjie) [inline notebooks/csrnet-training.ipynb:16]: <p>What is the reserved part for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CS3244-AY2021-SEM-1/cnn-model/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (teojunjie) [inline notebooks/csrnet-training.ipynb:16]: <p>Is this intentional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CS3244-AY2021-SEM-1/cnn-model/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (teojunjie) [inline notebooks/csrnet-training.ipynb:16]: <p>Not a big issue but this should be in evaluate model .py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CS3244-AY2021-SEM-1/cnn-model/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (benntay1) [reply notebooks/csrnet-training.ipynb:16]: I think its just the renaming of the function. if not will have FutureWarning if I'm not wrong

Comment (benntay1) [reply notebooks/csrnet-training.ipynb:16]: <p>oh pls change the learning rate and epochs accordingly</p>

Comment (benntay1) [reply notebooks/csrnet-training.ipynb:16]: <p>ok will make the addition into that .py file</p>

**Review by teojunjie — APPROVED** [review]

**Review by natalie-lam — APPROVED** [review]
